### ISSUE TITLE
Very low preplexity

### ISSUE THREAD

**Issue by gidim** [Marsan-Ma/tf_chatbot_seq2seq_antilm#12]
Thanks for updating to TF 1.0 !

Since the update i'm getting a really low perplexity score (6) starting from my first epoch.
I manage to minimize to 1 both on the train on dev set. When examining the actual output the results are really bad.

Is it possible that something is wrong with the loss function computation?

**Comment by Marsan-Ma** [comment]
Hi gidim,

Are you modifying the script `go_example` to do the training?
If yes, you've downloaded my pre-trained model, thus the perplexity is very low since it's already trained over 200k iterations.

As for the actual output, have you changed any parameter? or changed corpus, dictionary... 
anything might be caused by the downloaded files overwrite your original environment?

**Comment by gidim** [comment]
Hi Marsan,
This was actually my mistake, that's why i closed the bug.
Thanks for your help
